internal/mode/static/state/dataplane/configuration.go

```diff
@@ -907,9 +907,9 @@ func buildLogging(g *graph.Graph) Logging {
 	return logSettings
 }
 
-func GetDefaultConfiguration(configVersion int) Configuration {
+func GetDefaultConfiguration(g *graph.Graph, configVersion int) Configuration {
 	return Configuration{
 		Version: configVersion,
-		Logging: Logging{ErrorLevel: defaultErrorLogLevel},
+		Logging: buildLogging(g),
```

Comment:
>Also fixed an issue where the log level was not honored when no Gateway resources existed.

Hmm I know that this was a specific decision to do, what's the reason to honor the log level when any of these are true `g.GatewayClass == nil || !g.GatewayClass.Valid || g.Gateway == nil`

Reply:
Because even if a user hasn't created a Gateway yet doesn't mean we shouldn't honor the log level they want.

Reply:
makes sense 👍